tests/geometry/test_parameterisations.py

```diff
@@ -118,14 +118,6 @@ def test_error(self):
         with pytest.raises(GeometryParameterisationError):
             PrincetonD._princeton_d(10, 3, 0)
 
-    def test_bad_behaviour(self):
-        p = PrincetonD()
-        with pytest.raises(OptVariablesError):
-            p.variables.add_variable(BoundedVariable("new", 0, 0, 0))
-
-        with pytest.raises(OptVariablesError):
-            p.variables.remove_variable("x1")
```

Comment:
Is this implicitly not possible any more? It shouldnt probably be possible either way

Reply:
Ya that method does not exist anymore. Even if an attribute was added at runtime, the __iter__ for a frame uses the dataclass fields and most methods utilse that iter so I would yes it's implicitly not possible anymore.